packages/datatrak-web/src/routes/Routes.tsx

```diff
@@ -97,6 +109,14 @@ export const Routes = () => {
             <Route path={ROUTES.VERIFY_EMAIL_RESEND} element={<VerifyEmailResendPage />} />
           </Route>
         </Route>
+        <Route
```

Comment:
I think it would be nice to re-organise this a bit. Could we move it into the private routes section above so that we don't need to wrap it in the private route.